about.html

```diff
@@ -0,0 +1,173 @@
+          <h2 class="hero-heading-major">
+            TECHLive GLOBAL SUMMIT <br />
+            ABUJA, 2022
+          </h2>
+          <div class="hero-paragraph">
+            <p>
+              This year, 2022, the main issue of discussion is Software Error.
+              To Err is Human, To really Screw things up you need software and
+              as thus, we at Techlive bring together ideas, innovation, concepts
+              in handling software error from project conceptualization to
+              Project Deployment and maintenance.
+            </p>
+          </div>
+          <p class="hero-loc">
+            Please contact use Per Email, for any further questions about
+            TechLive Summit 2022!
+          </p>
+          <a class="hero-mail" href="http/email.com">techlive@istech.io</a>
+        </div>
+      </section>
+      <section class="about-logo-section">
+        <div class="about-logo-header">
+          <h2 class="header-text">
+            About TechLive <br />
+            Summit Logo
+          </h2>
+          <div class="header-line"></div>
+        </div>
+        <div class="logo-container">
+          <p class="logo-description">
+            The Logo of TechLive Summit 2022 was decided through the logo
+            competition from 28th July to 29th July
+          </p>
+          <div class="event-logo">
+            <img
+              class="event-logo-img"
+              src="./asset/images/techLiveLogo.png"
+              alt="event-logo"
+            />
+          </div>
+        </div>
+      </section>
+      <section class="past-summit-section">
+        <div class="past-summit-header">
+          <h2 class="header-text">See the Past Global TECHLive Summits</h2>
+          <div class="header-line"></div>
+        </div>
+        <div class="past-summit-container">
+          <p class="past-parag">
+            Take a look at the last two TECHLive Summit that took place at Lagos
+            and Port Harcourt.
+          </p>
+          <div class="picture-event-container">
+            <div class="past-event">
+              <img
+                src="./asset/images/about/past-summit-1.jpg"
+                alt="past event image"
+                class="event-img"
+              />
+              <div class="color-overlay">
+                <p>
+                  2020 <br />
+                  LAGOS
+                </p>
+              </div>
+            </div>
+            <div class="past-event">
+              <img
+                src="./asset/images/about/tech-event-2.jfif"
+                alt="past event image"
+                class="event-img"
+              />
+              <div class="color-overlay">
+                <p>
+                  2021 <br />
+                  PORT HARCOURT
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer class="about-footer-section">
```

Comment:
- [ ] @uisendy great work implementing the changes, kindly ensure the footer background in desktop view is in black as previously requested.

Reply:
Thanks for the review. That's covered now. Thanks